### DISCUSSION TITLE
Premium License Confusion

### DISCUSSION THREAD

**Discussion by AdityaKrishnamurthy** [aclap-dev/video-downloadhelper#1488] [Help and Q&A] [answered]
### Your question

As i am planning to buy the premium licence why are there different prices for the firefox and chrome licence?
And the firefox license price is way higher compared to chrome?
Can i know the differences of these 2 licences?
Also it does mean that if i get the chrome licence i won't be able to use it on firefox right?
Which license would be better?
Please explain me the differences to make a proper decision
Thank you

### Addon & CoApp version and details

![1001 - Video_DownloadHelper_-_Get_a_license_-_Google_Chro](https://github.com/user-attachments/assets/4b180dab-cbf3-4a04-b0b5-e9967dd8c585)
![1002 - Video_DownloadHelper_-_Get_a_license_—_Zen_Browser](https://github.com/user-attachments/assets/dbeae9b2-554e-419b-91d0-3197fec10cc5)

**Comment by jcfrog** [comment, marked as answer]
Hi,
You can't download from Youtube with Chrome extension, Google does not allow it.

**Comment by zirakabd** [comment]
I just purchased the key and I want to know How to change the key to firefox.

**Reply by tiagovix2000** [reply]
I'd like to know the same. I was not aware about the difference between Chrome and Firefox. For sure Firefox is better because has not the youtube issue. I'd like to change my license.

**Comment by jcfrog** [comment]
@zirakabd I sent you an email

@tiagovix2000 please send me an email at licenses@downloadhelper.net
